charms/kfp-api/tests/integration/test_charm.py

```diff
@@ -61,26 +63,52 @@ async def test_prometheus_grafana_integration(ops_test: OpsTest):
     """Deploy prometheus, grafana and required relations, then test the metrics."""
     prometheus = "prometheus-k8s"
     grafana = "grafana-k8s"
+    prometheus_scrape_charm = "prometheus-scrape-config-k8s"
+    scrape_config = {"scrape_interval": "5s"}
 
     await ops_test.model.deploy(prometheus, channel="latest/beta")
     await ops_test.model.deploy(grafana, channel="latest/beta")
     await ops_test.model.add_relation(prometheus, grafana)
     await ops_test.model.add_relation(APP_NAME, grafana)
-    await ops_test.model.add_relation(prometheus, APP_NAME)
+    await ops_test.model.deploy(
+        prometheus_scrape_charm,
+        channel="latest/beta",
```

Comment:
Why do we use latest/beta here?  This sounds like maybe we need a feature that'll be promoted later?  If yes, add a comment here and link an issue in the repo saying when we can remove this and pull from latest/stable again.  

In that issue, add a note to do the same for the testing instructions readme, too :)

Reply:
This charm is not available in stable. @mmanciop please correct me if I'm wrong, but I think latest/beta is currently the only channel it can be fetched from?

Reply:
Let's not hold merge up on this.  We can keep this and merge, but just add a comment/gh issue that reminds us to update when there's a stable version

Reply:
Sure, I created a gh issue for this. Thanks